src/Modules/DashboardModule/components/Dashboard/Dashboard.tsx

```diff
@@ -1,5 +1,209 @@
-import React from "react";
+import axios from "axios";
+import { useEffect, useState } from "react";
+import { Bar, Doughnut } from "react-chartjs-2";
+import { useUser } from "../../../../Context/AuthContext";
+import {
+  baseUrl,
+  getRequestHeaders,
+  handleApiError,
+} from "../../../../utils/Utils";
+import Header from "../../../SharedModule/components/Header";
+import {
+  Chart as ChartJS,
+  CategoryScale,
+  LinearScale,
+  BarElement,
+  DoughnutController,
+  ArcElement,
+  Title,
+  Tooltip,
+  Legend,
+} from "chart.js";
+
+import "./Dashboard.Moule.css";
+
+ChartJS.register(
+  CategoryScale,
+  LinearScale,
+  BarElement,
+  DoughnutController,
+  ArcElement,
+  Title,
+  Tooltip,
+  Legend
+);
 
 export default function Dashboard() {
-  return <>Dashboard</>;
+  const { userRole } = useUser();
+  const [todoCount, settodoCount] = useState<number>(0);
```

Comment:
There is no need for all states, we can have one and the others will be derived from it. This will make our code simpler.